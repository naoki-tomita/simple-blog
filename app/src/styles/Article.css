h2 {
  border-bottom: solid 1px #bbb;
}

.body {
  box-shadow: 0 4px 4px #ccc;
  margin-bottom: 16px;
  border-radius: 12px;
  min-height: 3em;
  padding: 12px 18px;
}

.body::first-letter {
  color: orange;
  font-size: 3em;
  line-height: 1;
  float: left;
}
