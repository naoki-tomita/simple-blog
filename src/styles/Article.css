h1 {
  padding-left: 42px;
}

p {
  padding-left: 52px;
}
