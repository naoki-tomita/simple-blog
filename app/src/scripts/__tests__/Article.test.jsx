import React from 'react';
import renderer from 'react-test-renderer';
import { Article } from '../Article';

describe("It should render with no crashing.", () => {
  it("should be render.", () => {
    expect(renderer.create(
      <Article title="title" body="body" />
    )).toMatchSnapshot();
  });
});
