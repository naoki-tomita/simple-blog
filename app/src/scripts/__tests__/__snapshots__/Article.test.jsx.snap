// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render with no crashing. should be render. 1`] = `
<div
  className="article"
>
  <h2>
    title
  </h2>
  <div
    className="body"
  >
    body
  </div>
</div>
`;
